core/src/components/select/test/a11y/select.e2e.ts

```diff
@@ -12,3 +12,30 @@ configs({ modes: ['ios'], directions: ['ltr'] }).forEach(({ title, config }) =>
     });
   });
 });
+
+configs({ directions: ['ltr'] }).forEach(({ title, config, screenshot }) => {
+  test.describe(title('select: a11y'), () => {
+    test.describe(title('select: font scaling'), () => {
+      test('should scale text on larger font sizes', async ({ page }) => {
+        await page.setContent(
+          `
+            <style>
+              html {
+                font-size: 310%;
+              }
+            </style>
+            <ion-select label="My Label" value="oranges" placeholder="Placeholder">
```

Comment:
Can you put this in a wrapper with a width that makes the text not truncate? Right now I believe it's getting scaled down to the smaller width possible